build/chord-min.js

```diff
@@ -1 +1 @@
anvas.style+="; transform: scaleX(-1);"),this.canvasSettings.reflect.vertical&&(this.canvasSettings.canvas.style+="; transform: scaleY(-1);")}},{key:"getCurrentTemplete",value:function(){return'\n    <div class="main-container">\n        <div class="accord-description">\n            <div class="description-element">\n                '.concat(this.customAttributes.currentChord.name,'\n            </div>\n            <div class="description-element row-number-container">\n                ').concat(this.customAttributes.currentChord.startString,'\n                </div>\n        </div>\n        <canvas\n            class="canvas-style"\n            width=').concat(this.canvasSettings.canvasWidth,"\n            height=").concat(this.canvasSettings.canvasHeight,"/>\n    </div>                                    \n    <style>\n        .accord-description {\n            display: flex;\n            justify-content: space-between;\n            border: solid ").concat(e.styleConstants.colors.borderColor[this.customAttributes.theme]," 1px;\n            border-bottom: none;\n            font-size: ").concat(e.styleConstants.fontSize.normal*this.customAttributes.size,"px;\n            color: ").concat(e.styleConstants.colors.textColor[this.customAttributes.theme],";\n            background-color: ").concat(e.styleConstants.colors.backgroundColor[this.customAttributes.theme],";\n        }\n        .description-element {\n            display: inline-block;\n            width: 50%;\n            padding-top: 2px;\n            padding-left: 5px;\n        }\n            .row-number-container {\n            display: inline-block;\n            text-align: right;\n            padding-right: 13px;\n        }\n        .main-container {\n            width: auto;\n            display: inline-block;\n            margin: 20px;\n            color: ").concat(e.styleConstants.colors.borderColor[this.customAttributes.theme],";\n        }\n        .canvas-style {\n            border: solid ").concat(e.styleConstants.colors.borderColor[this.customAttributes.theme]," 1px;\n            border-top: none;\n        }\n    </style>")}},{key:"makeIdEnding",value:function(){for(var t="-",i="ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789",n=0;n<5;n++)t+=i.charAt(Math.floor(Math.random()*i.length));return(new Date).getTime()+t}}])&&o(n.prototype,s),u&&o(n,u),i}();customElements.define("single-chord",l)},{"./chord-list":1,"./style-constants":3}],3:[function(t,i,n){"use strict";Object.defineProperty(n,"__esModule",{value:!0}),n.sizeList=n.styleConstants=void 0;n.sizeList={medium:1,large:2,small:.9};n.styleConstants={fontSize:{normal:14},colors:{circleColor:{dark:"#FFFFFF",light:"#000000"},basisColor:{dark:"#795548",light:"#E5CE8C"},stringsColor:{dark:"#cddc39",light:"#6D5454"},rowDividerColor:{dark:"#000000",light:"#FFFFFF"},borderColor:{dark:"#FFFFFF",light:"#000000"},textColor:{dark:"#FFFFFF",light:"#000000"},backgroundColor:{dark:"#000000",light:"#FFFFFF"}}}},{}]},{},[2]);
```

Comment:
shouldn't *min.js files be ignored by the git?

Reply:
it is necessary if you want to use just min.js without installing package by npm